docs/user/dql/functions.rst

```diff
@@ -1307,19 +1308,19 @@ Usage: date_format(date, format) formats the date argument using the specifiers
    * - x
      - x, for any smallcase/uppercase alphabet except [aydmshiHIMYDSEL]
 
-Argument type: STRING/DATE/DATETIME/TIMESTAMP, STRING
+Argument type: STRING/DATE/DATETIME/TIME/TIMESTAMP
```

Comment:
Second arg is STRING only I suppose
Can you return it back to follow the doc format?

Reply:
Fixed in https://github.com/Bit-Quill/opensearch-project-sql/pull/204/commits/c44602293107b2037d35f9c09138451a73c3bff1